### ISSUE TITLE
Slider value cannot be updated when VoiceOver is enabled

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#3033]
**Describe the bug**
Enabling the **VoiceOver** accessibility feature on iOS devices does not allow dragging the Slider (following the steps for [scrolling an iOS device](https://www.imore.com/how-use-voiceover-iphone-and-ipad#activate)). 
Singe-tap on the slider read the scale numbers.

When side buttons are enabled, the value can be changed, but the buttons are read as '_unreadable_'.

**To Reproduce**
https://stackblitz.com/edit/angular-wk4wyc?file=app/app.component.ts

**Browser**
- Safari
